test-workflow.yml

```diff
@@ -25,14 +25,39 @@ jobs:
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: us-west-2
 
-    - name: Render Amazon ECS task definition
-      id: render-task-definition
-      uses: aws-actions/amazon-ecs-render-task-definition@v1
+    - name: Render Amazon ECS task definition with task definition file 
+      id: render-task-definition-file
+      uses: shesaave/amazon-ecs-render-task-definition@v1
       with:
         task-definition: task-definition.json
         container-name: sample-app
         image: amazon/amazon-ecs-sample:latest
 
+    - name: Render Amazon ECS task definition with task definition arn (amazon resource name)
+      id: render-task-definition-arn
+      uses: shesaave/amazon-ecs-render-task-definition@v1
+      with:
+        task-definition-arn: task-definition-arn
+        container-name: sample-app
+        image: amazon/amazon-ecs-sample:latest
+
+    - name: Render Amazon ECS task definition with task definition family 
+      id: render-task-definition-family
+      uses: shesaave/amazon-ecs-render-task-definition@v1
+      with:
+        task-definition-family: task-definition-family
+        container-name: sample-app
+        image: amazon/amazon-ecs-sample:latest
+
+    - name: Render Amazon ECS task definition with task definition family and revision
+      id: render-task-definition-family-revision
+      uses: shesaave/amazon-ecs-render-task-definition@v1
+      with:
+        task-definition-family: task-definition-family
+        task-definition-revision: 1
+        container-name: sample-app
+        image: amazon/amazon-ecs-sample:latest
+
     - name: Validate Amazon ECS task definition
       env:
         TASK_DEF_FILE: ${{ steps.render-task-definition.outputs.task-definition }}
```

Comment:
Let's make sure that the changes in Validate and Register steps are reflected in both files. This seems to be **not** using the updated id of the steps from `render-task-definition` to `render-task-definition-file`.